astroquery/nasa_exoplanet_archive/tests/test_nasa_exoplanet_archive_remote.py

```diff
@@ -14,124 +14,159 @@
+    params = NasaExoplanetArchive.query_planet('HD 209458 b ')
+    assert_quantity_allclose(params['pl_radj'], 1.320 * u.Unit('R_jup'),
+                             atol=0.1 * u.Unit('R_jup'))
 
-    for planet in expected_planets:
-        assert planet.lower().replace(" ", "") in table['NAME_LOWERCASE']
 
-    assert 'pl_trandep' not in table.colnames
+@remote_data
+@pytest.mark.skipif('not APY_LT12')
+def test_hd209458b_exoplanets_archive_apy_gt12():
+    # Testing intentionally un-stripped string:
+    with pytest.raises(ValueError):
+        params = NasaExoplanetArchive.query_planet('HD 209458 b ')
+        assert_quantity_allclose(params['pl_radj'], 1.320 * u.Unit('R_jup'),
+                                 atol=0.1 * u.Unit('R_jup'))
 
 
 @remote_data
-def test_hd209458b_exoplanet_archive():
-    # Testing intentionally un-stripped string, no spaced:
-    params = NasaExoplanetArchive.query_planet('HD209458b ')
+def test_hd209458b_exoplanet_archive_coords():
+    params = NasaExoplanetArchive.query_planet('HD 209458 b ')
+    simbad_coords = SkyCoord(ra='22h03m10.77207s', dec='+18d53m03.5430s')
 
-    assert str(params['pl_hostname']) == 'HD 209458'
-    assert_quantity_allclose(params['pl_orbper'], 3.52474859 * u.day,
-                             atol=1e-5 * u.day)
+    sep = params['sky_coord'][0].separation(simbad_coords)
 
-    assert not params['pl_kepflag']
-    assert not params['pl_ttvflag']
+    assert abs(sep) < 5 * u.arcsec
 
 
-@remote_data
 @pytest.mark.skipif('APY_LT12')
-def test_hd209458b_exoplanets_archive_apy_lt12():
+def test_hd209458b_exoplanets_archive_apy_lt12_local():
     # Testing intentionally un-stripped string:
-    params = NasaExoplanetArchive.query_planet('HD 209458 b ')
+    params = NasaExoplanetArchive.query_planet('HD 209458 b ', table_path=LOCAL_TABLE_PATH)
     assert_quantity_allclose(params['pl_radj'], 1.320 * u.Unit('R_jup'),
                              atol=0.1 * u.Unit('R_jup'))
 
 
-@remote_data
 @pytest.mark.skipif('not APY_LT12')
-def test_hd209458b_exoplanets_archive_apy_gt12():
+def test_hd209458b_exoplanets_archive_apy_gt12_local():
     # Testing intentionally un-stripped string:
     with pytest.raises(ValueError):
-        params = NasaExoplanetArchive.query_planet('HD 209458 b ')
+        params = NasaExoplanetArchive.query_planet('HD 209458 b ', table_path=LOCAL_TABLE_PATH)
         assert_quantity_allclose(params['pl_radj'], 1.320 * u.Unit('R_jup'),
                                  atol=0.1 * u.Unit('R_jup'))
 
 
-@remote_data
-def test_hd209458b_exoplanet_archive_coords():
-    params = NasaExoplanetArchive.query_planet('HD 209458 b ')
+def test_hd209458b_exoplanet_archive_coords_local():
+    params = NasaExoplanetArchive.query_planet('HD 209458 b ', table_path=LOCAL_TABLE_PATH)
     simbad_coords = SkyCoord(ra='22h03m10.77207s', dec='+18d53m03.5430s')
 
-    sep = params['sky_coord'].separation(simbad_coords)
+    sep = params['sky_coord'][0].separation(simbad_coords)
```

Comment:
@keflavich In this new change, we have to do the indexing to get the data. Is that okay?

Reply:
Yes, that's not a problem, particularly in the tests.  It might be nice to have a comment telling why we're indexing and what the underlying data structure is, but it's not necessary.